### ISSUE TITLE
Improve output rendering

### ISSUE THREAD

**Issue by joyceerhl** [joyceerhl/vscode-github-graphql-notebooks#3]
Right now everything is just displayed using the builtin `application/json` mimetype renderer

**Comment by jason-dour** [comment]
Looking to leverage the extension to assist with prototyping queries.  But the output type of `text/x-json` renders as black text, which on a dark mode theme makes the output difficult to read.

I changed the extension locally, modifying [line 226](https://github.com/joyceerhl/vscode-github-graphql-notebooks/blob/9798dd8469c5f813097b1729443ae6649e2f1884/src/extension.ts#L226) to use `application/json` instead of `text/x-json` and the output now formats and highlights like any other JSON and is legible since it follows standard highlighting/theme.

**Comment by joyceerhl** [comment]
The switch to `text/x-json` is because VS Code is moving support for `application/json` to a third-party extension. @rebornix this sounds like a potential bug in the `rebornix.vscode-code-renderer` extension? I also noticed the renderer extension is not web-enabled, which makes the GitHub GraphQL extension unusable in web.

**Comment by rebornix** [comment]
@joyceerhl there would be two types of code highlight support.

The first one is shipped with VS Code, it uses the markdown renderer to highlight the code, like a code block. It has right colors and tokenizations and it should be web enabled.

The catch with first option is it's not an editor, so no folding support. That's when my code renderer extension can be useful.

**Comment by jason-dour** [comment]
@joyceerhl thanks for the explanation. I spent a fair amount of time trying to figure out why `text/x-json` was not highlighting.  Is that something I can tune somewhere (missing config)?

Web compatibility would be extremely nice as well.  :)

**Comment by jason-dour** [comment]
Indeed!  It's web version compatible and renders highlighting just fine.  Thanks!  :)
